crosswars-backend/src/main/kotlin/xyz/crosswars/service/GroupService.kt

```diff
@@ -109,4 +113,11 @@ class GroupService(
         isMemberRepository.save(savedIsMember)
         return savedIsMember
     }
+
+    private fun isGroupNameValid(group_name: String): Boolean {
+        if(group_name.length < 3 || group_name.length > 25) return false
+        //match with alphanumeric, separated by space between words
+        val pattern = "^[a-zA-Z1-9]+( ?[a-zA-Z-1-9])*$".toRegex()
```

Comment:
Extract this pattern variable out and give it a descriptive constant name like `VALID_GROUP_NAME_PATTERN`.